### DISCUSSION TITLE
Applying Theme Variant Class in the `<html>`

### DISCUSSION THREAD

**Discussion by brunoaugusto** [RyanClementsHax/tailwindcss-themer#61] [Q&A]
I'm porting a theme created with Bootstrap to Tailwind, and I came across an issue regarding coloured scrollbars.

The general idea is to set `scrollbar-color` and their WebKit vendor prefixes to the overflowing content, and that works nicely with the current implementation described by this Plugin's docs. However, it seems that the only way to colourize the main scrollbars on Firefox is by setting the said rules to the `<html>` while for Chrome — and possibly other WebKit browsers — setting on `<body>` is enough.

Problem is, blocks like this:

```
html, body, textarea {
    @apply
      theme1:scrollbar-thumb-[#452524] theme1:scrollbar-track-[#241413]
        theme2:scrollbar-thumb-[#004500] theme2:scrollbar-track-[#032100]
          theme3:scrollbar-thumb-[#243659] theme3:scrollbar-track-[#121C32]
            theme4:scrollbar-thumb-[#047857] theme4:scrollbar-track-[#003D30]
              theme5:scrollbar-thumb-[#5E095E] theme5:scrollbar-track-[#2E052E]
                theme6:scrollbar-thumb-[#730000] theme6:scrollbar-track-[#4A0000]
                  theme7:scrollbar-thumb-[#70001C] theme7:scrollbar-track-[#003D30]
                    theme8:scrollbar-thumb-[#C8C8C8] theme8:scrollbar-track-[#191919]
                      scrollbar-thumb-[#6B6B6B] scrollbar-track-[#191919]; /** Default */
  }
```

> `scrollbar-thumb-*` and `scrollbar-track-*` are part of a 3rd-party plugin that simplifies the rules for scrollbar colouring

Would create rules like this:

````
.theme1 body, .theme1 html, .theme1 textarea ... {} /** Shortened for brevity */
````

Which, as much as I know, — and accordingly to Tailwind's docs about [manual Dark Mode](https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually) — includes an incorrectly parsed directive `.theme1 html` while it should be `html.theme1`.

I *could* have made an Issue of this, but I thought the best to discuss it first. If it's like I'm thinking, perhaps you could condition the variant expression based on the name/type of the tag if the selector is `html` or `*` (both are affected).

Thank you for your time

**Comment by RyanClementsHax** [comment]
That is a good feature request. Being able to configure how themes get applied is a common request. It is on my list of things to add when I prioritize the time.

**Reply by RyanClementsHax** [reply]
I released this in the most recent version of the plugin :)
